python/cudf/cudf/io/orc.py

```diff
@@ -30,6 +30,7 @@ def read_orc(
     columns=None,
     stripe=None,
     stripe_count=None,
+    stripes=None,
```

Comment:
Can we have just `stripes` and not support `stripe` / `stripe_count` anymore?

Reply:
Well, the C++ API already supported `stripe_list`, `stripe`, and `stripe_count` [1]. This PR is actually just exposing the `stripe_list`. If you still think `stripe`/`stripe_count` should be discontinued, should the change be made on this PR or a separate one? and should it be made only in the Python API or also the C++ API?

[1] https://github.com/rapidsai/cudf/blob/3cf6c25714e6d0b5ede8a80e6440b647b8f87c35/cpp/include/cudf/io/functions.hpp#L355-L360

Reply:
We should follow what we did on the parquet side, which I believe is remove the `stripe` and `stripe_count` APIs.

Reply:
Sounds good. So I removed `stripe` and stripe_count` from the Python API [1] and from the C++ API [2]. Let me know if this looks okay.

[1] https://github.com/rapidsai/cudf/pull/6024/commits/806b932bd221a05533455376dd44eeac27b4ed50
[2] https://github.com/rapidsai/cudf/pull/6024/commits/7f32886a3091601462d322479e7b9e78fceb3f55